OsuPlayer/Views/SettingsView.axaml

```diff
@@ -22,16 +22,18 @@
 
     <Grid RowDefinitions="Auto, *">
         <Grid Row="0">
-            <AutoCompleteBox Name="SettingsSearch" Text="{Binding SettingsSearchQ}" ItemsSource="{Binding SettingsCategories}"
+            <AutoCompleteBox Name="SettingsSearch" Text="{Binding SettingsSearchQ}"
+                             ItemsSource="{Binding SettingsCategories}"
                              VerticalAlignment="Center" HorizontalAlignment="Stretch"
-                             Watermark="type to search for settings" Margin="5" MaxWidth="1500" >
+                             Watermark="type to search for settings" Margin="5" MaxWidth="1500">
                 <AutoCompleteBox.ItemTemplate>
                     <DataTemplate x:DataType="Control">
                         <Panel>
                             <Grid ColumnDefinitions="*, Auto">
                                 <StackPanel Grid.Column="0" HorizontalAlignment="Left"
                                             VerticalAlignment="Center">
-                                    <TextBlock Text="{Binding Name}" Foreground="{DynamicResource AccentTextFillColorPrimaryBrush}" />
+                                    <TextBlock Text="{Binding Name}"
+                                               Foreground="{DynamicResource AccentTextFillColorPrimaryBrush}" />
```

Comment:
We probably have to check the project settings for why your formater did this. probably this stupid line thingy.

But is fine for now.